### PR TITLE
Frontend 1656 - Add Synaptic Header to Dev Center

### PR DESCRIPTION
**Pull request by langlk**
Resolves Dev Center side of [FRONTEND-1656](https://algorithmia.atlassian.net/browse/FRONTEND-1656)

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -1,11 +1,208 @@
...
+        <a class="hide-unauth" href="/">
+          <button
+            class="syn-btn outlined theme-primary syn-ml-16 syn-visible-lg"
+          >
+            Return to App
+          </button>
+        </a>
+      </div>
+    </div>
+    <div class="syn-page-header-nav" @click="toggleNavMenu">
```
Comment (bnlb) [inline _includes/header.html]: `@click` here.

**Comment by langlk** [comment]
> This looks great! I had to `cd` into the submodule and run `npm install && npm run build` before I could see the changes though. We'll have to make sure everything [here](https://github.com/algorithmiaio/synapse/pull/175#discussion_r328272904) is addressed before we deploy.

I've added a synapse build step to the Dockerfile for the dev center in FRONTEND-1657, so it should be fine to deploy. I'll be sure to add a note about building synapse before running the dev center locally

**Comment by langlk** [comment]
@bnlb Added instructions for local Synapse setup to the readme in #208

File: _includes/header.html (hunk 1 above)
Comment (langlk) [reply _includes/header.html]: Removed!

**Comment by andryanmiller** [comment]
Wondering if we should hide any dynamic elements until we know the user is auth'd? I'm seeing a flash of buttons on page load from unauth'd -> auth'd when logged in.

**Comment by langlk** [comment]
> Wondering if we should hide any dynamic elements until we know the user is auth'd? I'm seeing a flash of buttons on page load from unauth'd -> auth'd when logged in.

Yeah, I think that's a good call. Will update!

**Comment by langlk** [comment]
These changes are up on test right now

**Comment by andryanmiller** [comment]
Perhaps PEBKAC?

test algorithmia com_developers(iPad)

**Comment by peckjon** [comment]
I'm not seeing the same overlap Ryan is, but I am seeing a "sign in" button even when I'm signed in

**Comment by langlk** [comment]
I think the css is getting cached, try clearing that/loading in an incognito window?

**Comment by bnlb** [comment]
Accidentally added my earlier comment to the web PR.

Just noticed this menu doesn't close when an outside click happens.
Screen Shot 2019-09-27 at 12 41 56 PM

**Comment by langlk** [comment]
@bnlb dropdown is fixed!

@andryanmiller / @peckjon looks like this was a caching issue, created a [ticket](https://algorithmia.atlassian.net/browse/FRONTEND-1816) to add some cachebusting for dev center assets.

**Pull request merged**